Flickr/Resources/Assets.xcassets/Heart Filled.imageset/Contents.json

```diff
@@ -0,0 +1,23 @@
+{
+  "images" : [
+    {
+      "filename" : "Vector.png",
```

Comment:
Please ensure you rename these assets before adding them to the project. Please set the name to the same value as the Asset in Xcode -> `Heart Filled` in this case.